transformer_engine/common/include/transformer_engine/activation.h

```diff
@@ -26,7 +26,7 @@ extern "C" {
  *  \param[in]     stream    CUDA stream used for the operation.
  */
 
-enum class NVTE_Activation_Type {
+enum NVTE_Activation_Type {
```

Comment:
Good catch, this breaks the C API. That said, I think enums like `GELU` and `RELU` are too common and will likely to run into name conflicts. I don't see this used in the C API, so a better approach would be put this inside a `#ifdef __cplusplus`. In the future it may be better to take advantage of C++ features (put within the `transformer_engine` namespace and rename to `Activation_Type`), but that's beyond the scope of this PR.

Reply:
For now:
```suggestion
enum class NVTE_Activation_Type {
```